Example/ios/Example.xcodeproj/project.pbxproj

```diff
@@ -418,6 +434,14 @@
 			name = Products;
 			sourceTree = "<group>";
 		};
+		27886A691F901098009513EF /* Recovered References */ = {
+			isa = PBXGroup;
+			children = (
+				325FF470492B468AAFC9B057 /* libRNAdMobManager.a */,
+			);
+			name = "Recovered References";
+			sourceTree = "<group>";
+		};
```

Comment:
Can we avoid this "Recovered References"?

Reply:
oops didn't mean to commit this file. will remove